eZ/Publish/Core/Search/Common/FieldNameResolver.php

```diff
@@ -79,26 +82,7 @@ public function __construct(
      */
     protected function getFieldMap()
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/ea68a7308fbb2d49301821e454850b915d7c7bf0